### DISCUSSION TITLE
Suggestion: Image conversion

### DISCUSSION THREAD

**Discussion by DaleJ122** [CorentinTh/it-tools#1362] [Ideas]
I find myself often having to spin up image magick or using limited online tools to convert images to other formats, e.g. jpg to png, webp, avif. It would be great to have one on here that isn't as limited

**Comment by sharevb** [comment]
Hi @DaleJ122, implemented in #1328 and here: https://sharevb-it-tools.vercel.app/image-converter, if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)

**Reply by DaleJ122** [reply]
This is great thank you! 

A couple of issues. 

1. It's not completely obvious that it will automatically convert as soon as an image is selected, and the conversion type should probably be near the top. This caused me to convert something to png before realising I had to choose the type first.
2. The old extension is left on the name of the new file e.g. image1.jpg converted to png becomes image1.jpg.png

**Reply by sharevb** [reply]
Hi @DaleJ122, thanks for your reporting, a new version is available at https://sharevb-it-tools.vercel.app/image-converter (needs ctrl+f5) that also handle multiple file conversion at once
